strings.txt

```diff
@@ -26635,3 +26635,54 @@ SETUP_YEARMENU_1
 	NL	Toon jaartal van album en nummer
 	PT	Mostrar o ano do album e de cada pista
 
+SETUP_ALBUMLINKROLES
+	DE	Für diese Rollen Künstlerlinks zu Albumlisten hinzufügen
+	EN	Add artist links to album lists for these roles
```

Comment:
These must be shorter. Even English takes two lines... What about "Linked roles in album lists"?

Reply:
Or "Linked Artists in album lists" ?